packages/office-ui-fabric-react/src/components/Overlay/Overlay.tsx

```diff
@@ -2,13 +2,23 @@ import * as React from 'react';
 import {
   css,
   getNativeProps,
-  divProperties
+  divProperties,
+  setBodyScroll
 } from '../../Utilities';
 import { IOverlayProps } from './Overlay.Props';
 
 import styles from './Overlay.scss';
 
 export class Overlay extends React.Component<IOverlayProps, {}> {
+
+  public componentDidMount() {
+    setBodyScroll(false);
```

Comment:
@dzearing hm.. if I don't know what it's for `setBodyScroll(true)` doesn't really tell me that it's enabling/disabling scrolling.